tests/php/src/ReaderThemeLoaderTest.php

```diff
@@ -151,12 +151,29 @@ public function test_filter_wp_prepare_themes_to_indicate_reader_theme() {
 		AMP_Options_Manager::update_option( Option::READER_THEME, $reader_theme_slug );
 		$this->assertEquals( $active_theme_slug, get_stylesheet() );
 		$this->assertEquals( $reader_theme_slug, $this->instance->get_reader_theme()->get_stylesheet() );
-		wp_set_current_user( self::factory()->user->create( [ 'role' => 'administrator' ] ) );
+
+		if ( is_multisite() ) {
+			$user_id = self::factory()->user->create(
+				[
+					'role' => 'administrator',
+				]
+			);
+
+			grant_super_admin( $user_id );
+			wp_set_current_user( $user_id );
+		} else {
+			wp_set_current_user( self::factory()->user->create( [ 'role' => 'administrator' ] ) );
+		}
 
 		// Note that this is added via filter and not called directly because the filtered value is keyed by theme slug,
 		// but the return value of wp_prepare_themes_for_js() is keyed with numeric indices.
 		$this->instance->register(); // This adds a `wp_prepare_themes_for_js` filter.
-		$themes = wp_prepare_themes_for_js();
+		$themes = wp_prepare_themes_for_js(
+			[
+				wp_get_theme( $active_theme_slug ),
+				wp_get_theme( $reader_theme_slug ),
```

Comment:
Why were these needed to be added here explicitly I wonder?

Reply:
On single-site installation, it returns all themes in an array but in the multisite environment, it only returns the current installed theme and a default theme.  Maybe because in respect to the current blog it can only fetch the theme that is installed?